src/AppInstallerCommonCore/GroupPolicy.cpp

```diff
@@ -104,26 +104,110 @@ namespace AppInstaller::Settings
             // Use folding to call each policy validate function.
             (FoldHelper{}, ..., Validate<static_cast<ValuePolicy>(P)>(policiesKey, policies));
         }
+
+        // Reads a list from a Group Policy.
+        // The list is stored in a sub-key of the policies key, and each value in that key is a list item.
+        // Cases not considered by this function because we don't use them:
+        //  - When the list is in an arbitrary key, not a sub key.
+        //  - When the list values are mixed with other values and are identified by a prefix in their names.
+        //  - When the value names are relevant.

```

Comment:
How do these values end up looking?  Are the value names under the control of the IT Admin, but we simply don't care what they are?  Or are they named by the system?

Reply:
The ADMX definition gives two options: let the IT Admin name them, or let the system name them just numbering (0, 1, ..) with an optional prefix set by us.
The PR I have for the ADMX lets the IT Admin set the names but I may change it to automatic.
In any case we don't care about the value names.